lang/src/org/partiql/lang/util/UtilityExtensions.kt

```diff
@@ -1,25 +1,34 @@
-/*
- * Copyright 2019 Amazon.com, Inc. or its affiliates.  All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License").
- *  You may not use this file except in compliance with the License.
- * A copy of the License is located at:
- *
- *      http://aws.amazon.com/apache2.0/
- *
- *  or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
- *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
- *  language governing permissions and limitations under the License.
- */
```

Comment:
We still need a copyright header, don't we? Or has that changed?

Reply:
FWIW, Theo told me once that it wasn't necessary as long as there was a license file in the root of the repository.

Reply:
So I am opting to not include the license in these files.